### ISSUE TITLE
Can we introduce light and dark mode using color-mode-module?

### ISSUE THREAD

**Issue by rajesh-h** [bdrtsky/nuxt-dev-to-clone#4]
Hi @bdrtsky ,

Could you please introduce light/dark theme mode in the nuxt-dev-to-clone webapp using official module https://github.com/nuxt-community/color-mode-module

This will definitely help a lot and I hope others as well in future.

Thank you very much for considering my request.

**Comment by bdrtsky** [comment]
Great suggestion, I'll start implementing it.

**Comment by rajesh-h** [comment]
@bdrtsky do you think you will look into it sometime soon?

**Comment by bdrtsky** [comment]
Yes, sorry it took long, was little busy. I started to add dark mode, should be added soon!

**Comment by dosstx** [comment]
Did this ever happen? Love this tutorial you made, @bdrtsky !
